src/app/components/ScrollLock.tsx

```diff
@@ -13,11 +13,11 @@ const ScrollLockBodyClass = createGlobalStyle`
       ${theme.breakpoints.mobile(css`
         @media print {
           #root {
-            display: none;
+            display: block;
```

Comment:
i'm not sure what is up with this.. when i look in heroku the body is still getting `display: none` and it does seem necessary for making the highlights print screen display right. is there something else applying this style? is it necessary here?

Reply:
I think this was done in error after I reverted a change to the breakpoints.